src/components/CardList/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Card from '../CardList/Card';
+
+const Cards = props => props.cards.map(card => {
+    return <Card 
+              headerData={card.headerData} 
+              bodyData={card.bodyData} 
+              viewMode={props.viewMode}
+              key={card.id}
+              id={card.id}
```

Comment:
I would advice to pass the who card object with all props and then destruct it if needed - this will prevent your code from changes if new properties appear in the card entry.